contracts/contracts/GrantRegistry.sol

```diff
@@ -19,23 +19,28 @@ contract GrantRegistry {
 
   /// @notice Grant object
   struct Grant {
-    uint96 id; // grant ID, as uint96 to pack into same slot as owner (this implies a max of 2^96-1 = 7.9e28 grants)
+    // Slot 1 (within this struct)
+    // Using a uint96 for the grant ID means a max of 2^96-1 = 7.9e28 grants
+    uint96 id; // grant ID, as
```

Comment:
I agree with these integer size changes. I do wonder, though, if this one in particular is worth it. We're not saving a slot, so we might actually be adding gas costs (if this change is considered in isolation).

Reply:
Sorry not sure I follow? The change highlighted here is just a comment change, even prior to this PR we packed the ID and owner into the same slot